deployment/roles/reverseproxy/templates/security-headers.toml.j2

```diff
@@ -0,0 +1,35 @@
+[tls.options]
+    [tls.options.default]
+      minVersion = "VersionTLS12"
+      sniStrict = true
+      cipherSuites = [
+          "TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256",
+          "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256",
+          "TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384",
+          "TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305",
+          "TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305",
+          "TLS_AES_128_GCM_SHA256",
+          "TLS_AES_256_GCM_SHA384",
+          "TLS_CHACHA20_POLY1305_SHA256",
+          "TLS_FALLBACK_SCSV"
+      ]
+
+[http.middlewares]
+    [http.middlewares.security-headers.headers]
+      # CORS
+      AccessControlAllowMethods = ["GET", "OPTIONS", "PUT"]
+      AccessControlAllowOrigin = "origin-list-or-null"
+      AccessControlMaxAge = 100
+      #AddVaryHeader = true
+      BrowserXssFilter = true
+      ContentTypeNosniff = true
+      ForceSTSHeader = true
+      FrameDeny = true
+      SSLRedirect = true
+      STSIncludeSubdomains = true
+      STSPreload = true
+      ContentSecurityPolicy = "default-src 'self' https://validator.spaceapi.io/v1/validate/ https://use.fontawesome.com/releases/v5.8.1/; script-src 'self' 'unsafe-inline'"
```

Comment:
Again, we're setting CSP policies for a certain endpoint (the validator) on the entire server, right? Also, the fontawesome URL is terribly specific, if we'd upgrade the fontawesome release the page would break once deployed.

Reply:
I would leave it in for now and work on removing fontawesome from the website itself since it's a privacy violation in my regard.
Same goes for the script-src 'unsafe-inline'.

I would also set the validator to something less specific and e.g. enable default-src for the whole domain.

Reply:
I agree about self-hosting the font-awesome files.

I still think that CSP should be in the service configuration, and not in a generic include-this-for-security-headers include file. CSP is site specific, some services might support more restrictive policies, others might require more loose ones.

Reply:
For me, these are the default headers that the service should provide, everything else you should specify yourself and set the header at from your application, better in your service configuration.

Basically i want to have the CSP Line like this for everything:
```
ContentSecurityPolicy = "default-src 'self' https://*.spaceapi.io; script-src 'self'"
```

But right now the website won't work with it.
I've started to refactor the website, but it might take a while.